src/server/middleware/auth.ts

```diff
@@ -146,6 +158,8 @@ const requireEditAssessmentFile = async (req: Request, _res: Response, next: Nex
 
 export const AuthMiddleware = {
   requireEdit,
```

Comment:
is it true, requireEdit now it's not used anymore from outside ?

Reply:
it's used here:

-     express.post(ApiEndPoint.Area.country(), AuthMiddleware.requireEdit, postCountry)
-     express.post(ApiEndPoint.CycleData.OriginalDataPoint.one(), AuthMiddleware.requireEdit, createOriginalDataPoint)
- express.delete(ApiEndPoint.CycleData.OriginalDataPoint.one(), AuthMiddleware.requireEdit, deleteOriginalDataPoint)
- express.get(ApiEndPoint.CycleData.OriginalDataPoint.one(), AuthMiddleware.requireView, getOriginalDataPoint)
- express.put(ApiEndPoint.CycleData.OriginalDataPoint.one(), AuthMiddleware.requireEdit, updateOriginalDataPoint)

Reply:
can u use requireEditTableData and make requireEdit internal to AuthMiddleware.requireEdit?

Reply:
done